spec/lib/whitehall_importer/create_image_revision_spec.rb

```diff
@@ -86,6 +86,19 @@
       end
     end
 
+    context "Image has exif data" do
+      let(:whitehall_image) do
+        build(:whitehall_export_image, fixture_file: "960x640-rotated.jpg")
+      end
+
+      it "should strip the exif data from the image" do
+        revision = described_class.call(document_import, whitehall_image, ["valid-image.jpg"])
```

Comment:
I wonder if it's worth checking for the present of exif data before processing the image in order to be certain it has been stripped at the end?

Reply:
Hmm I know what you mean but you can't really as the image doesn't exist before the point of execution. I mean you could test the image fixture but you shouldn't need to because you should trust a fixture is what it says it is.

I pondered whether a test is even needed as you're pretty much just testing methods provided by minimagick but decided we should probably have one. :)

Reply:
Sure, is it worth renaming the image to specify that it has exif data?  I suppose my concern is if the fixture somehow gets replaced with a image that doesn't have it (if that's even possible)...I don't know much about exif to be honest 🤷‍♀

Reply:
I mean I can rename to include exif but the 'rotate' implies that it has exif orientation data and it also follows the naming convention of another fixture that presumably also contains exif orientation data - '640x960-rotated.jpg' so I think the naming is okay unless you still think it's not clear?